editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Node.tsx

```diff
@@ -66,6 +66,8 @@ const Node: React.FC<any> = (props) => {
       return <Question {...allProps} text="Find property" />;
     case TYPES.List:
       return <Question {...allProps} text={node?.data?.title ?? "List"} />;
+    case TYPES.MapAndLabel:
+      return <Question {...allProps} text="Map and Label" />;
```

Comment:
nit: once you add a `title` field in the Editor, then we could follow the pattern `text={node?.data?.title ?? "Map and label"} />` - typically makes nodes much easier to navigate in the graph if using custom title rather than generic component name !